Resources/Audio/Weapons/Xeno/attributions.yml

```diff
@@ -0,0 +1,9 @@
+- files: ["alien_claw_flesh1", "alien_claw_flesh2", "alien_claw_flesh3"]
+  license: "CC-3.0-BY-SA"
+  copyright: "Taken from CM-SS13"
+  source: "https://gitlab.com/cmdevs/colonial-warfare/-/tree/dev/sound/weapons"
+
+- files: ["alien_spitacid"]
+  license: "AGPL-3.0"
+  copyright: "Taken from tgstation"
```

Comment:
these should be CC-BY-NC-SA-3.0 from tgmc

Reply:
Changed it